torch/nn/parallel/distributed.py

```diff
@@ -1205,7 +1205,6 @@ def _passing_sync_batchnorm_handle(self, module_copies):
                     assert (
                         self.device_type != "cpu"
                     ), "SyncBatchNorm layers only work with GPU modules"
-                    layer._specify_ddp_gpu_num(1)
```

Comment:
if I understand it correctly, _specify_ddp_gpu_num(1) is kind of passing handle to syncBatchNorm, letting syncBatchNorm know that the syncBatchNorm is working with DDP.

Reply:
This really depends on whether we want to remove `ddp_gpu_size` field, which will always be 1 if not None.

Do you think we should still keep this field?